CONTRIBUTING.md

```diff
@@ -21,6 +21,10 @@ Please report unacceptable behavior to the nmstate team.
 - Code: Managed on [GitHub](https://github.com/nmstate/nmstate) through
   Pull Requests ([PR](https://github.com/nmstate/nmstate/pulls)).
 
+#### Running tests
```

Comment:
How about adding this to the first item for the next section about pull requests?

Reply:
Sure, done!